BrentOzarULTD/SQL-Server-First-Responder-Kit
# Issue 407

Fixes #407

Changes proposed in this pull request:
 - Split the what's running now into it's own stored procedure
 - Modified that script into a version runnable against any recent version
 - Altered sp_BlitzFirst to call this code

How to test this code:
 - Make sure sp_BlitzFirst installs correctly
 - Make sure sp_BlitzFirst runs with no errors if sp_BlitzWho is not installed (there should be a note printed, but no errors)
 - Make sure sp_BlitzWho installs correctly
- Make sure sp_BlitzWho runs with no errors and returns an expected dataset

Has been tested on (remove any that don't apply):
 - Case-sensitive SQL Server instance (2008)
 - SQL Server 2008
 - SQL Server 2014
 - SQL Server 2016
